docs-2.0/3.ngql-guide/1.nGQL-overview/1.overview.md

```diff
@@ -94,10 +94,10 @@ The following are some major differences (by design incompatible) between nGQL a
 |Schema| Optional Schema | Strong Schema |
 |Equality operator| `=` |  `==` |
 |Math exponentiation| `^` | `^` not supported. Use pow(x, y) instead. |  
-|Edge rank| no such concept | edge rank (reference by @)|
+|Edge rank| No such concept. | edge rank (reference by @)|
 |Statement|-|All DMLs (`CREATE`, `MERGE`, etc) of openCypher 9, and `OPTIONAL MATCH` are not supported. |
 |Label and tag| A label is used for searching a vertex, namely an index of vertex. | A tag defines the type of a vertex and its corresponding properties. It cannot be used as an index. |
-| Pre-compiling and parameterized query  | support | not supported |
+| Pre-compiling and parameterized query  | Support | Only parameterized query is supported. |
```

Comment:
better use "only the parameterized query is supported" or "only parameterized queries are supported" in the detailed description.